src/SourceBuild/tarball/content/repos/roslyn.proj

```diff
@@ -28,14 +28,10 @@
          repository. This must be lower than MicrosoftNetCompilersToolsetVersion,
          but not higher than our minimum dogfoodable Visual Studio version, or else
          the generators we build would load on the command line but not load in IDEs.
-      In source-build these don't need to be pinned and can use the source-built versions since it doesn't
-      need to support VS.
+
+      In source-build we use ref-only package from SBRP.
     -->
-    <ExtraPackageVersionPropsPackageInfo Include="SourceGeneratorMicrosoftCodeAnalysisVersion" Version="%24(MicrosoftCodeAnalysisCommonVersion)" />
-    <ExtraPackageVersionPropsPackageInfo Include="SourceBuildLiftedSystemCollectionsImmutableVersion" Version="%24(SystemCollectionsImmutableVersion)" />
-    <ExtraPackageVersionPropsPackageInfo Include="SourceBuildLiftedSystemReflectionMetadataVersion" Version="%24(SystemReflectionMetadataVersion)" />
-    <ExtraPackageVersionPropsPackageInfo Include="SourceBuildLiftedSystemRuntimeCompilerServicesUnsafeVersion" Version="%24(SystemRuntimeCompilerServicesUnsafeVersion)" />
-    <ExtraPackageVersionPropsPackageInfo Include="SourceBuildLiftedSystemTextEncodingCodePagesVersion" Version="%24(SystemTextEncodingCodePagesVersion)" />
+    <ExtraPackageVersionPropsPackageInfo Include="SourceGeneratorMicrosoftCodeAnalysisVersion" Version="$(RefOnlyMicrosoftCodeAnalysisCommonVersion)" />
```

Comment:
Is this ExtraPackageVersionPropsPackageInfo property even needed anymore?  If we don't override it, it looks like roslyn's value would get picked up which is 4.1.0 as well.  If we define our own value, it will likely never get updated if roslyn takes a dependency on a different version.

Reply:
If we let Roslyn set version, wouldn't that introduce a prebuild once they move forward?

Reply:
But, that's probably better as it will prompt us to add a new sbrp package.

Reply:
That's correct - we would like source-build to build with the same version as the Microsoft build.